### PR TITLE
Added a new dependency to install Grid driver on Linux.

### PR DESCRIPTION
**Pull request by ahkhaldi**

### CODE REVIEW

File: src/VirtualClient/VirtualClient.Dependencies/GridDriverInstallation.cs [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                    throw new DependencyException("GRID driver installation is not supported on Windows");
+                }
+
+                VirtualClientRuntime.IsRebootRequested = this.RebootRequired;
+            }
+
+            this.Logger.LogTraceMessage($"{this.TypeName}.ExecutionCompleted", telemetryContext);
+        }
+
+        private async Task<bool> CheckIfDriverInstalledAsync(EventContext telemetryContext, CancellationToken cancellationToken)
```
Comment (LiYueqian-James) [inline src/VirtualClient/VirtualClient.Dependencies/GridDriverInstallation.cs:149]: can we log the driver version as a metric as well?

**Comment by yangpanMS** [comment]
Please help add the documents to website/docs/dependency to explain very basically what it does, and an example json snippet

**Comment by yangpanMS** [comment]
Ok thanks, could you please help pull latest main and make sure PR builds? Then I will take a look

**Comment by yangpanMS** [comment]
Hey could you abandon this if this is covered in the new PR?

**Pull request closed without merging**